PUBLISH.md

```diff
@@ -57,6 +57,15 @@ Once you have done that, the following command will publish the latest version
 npm --registry=https://registry.npmjs.com publish
 ```
 
+5. **Updating github**
+Be sure to push the right tag corresponding to the npm release version to github.
+
+```bash
+git push origin 6.0.0 # replace 6.0.0 with the latest version.
```

Comment:
I think this command could be replaced with `git push --follow-tags` since the `make release-*` command should create the new tag w/ the new commit.